packages/components/stepper/src/js/step-form/step-form.controller.js

```diff
@@ -54,6 +48,12 @@ export default class StepFormController {
       .addClass('oui-stepper__step oui-stepper__step_form')
       .removeAttr('id')
       .removeAttr('name'));
+
+    // Check if Stepper parent
+    if (this.stepperCtrl) {
```

Comment:
Is the component supposed to work without a `stepperCtrl` ? If not, send an error so the user is warned

Reply:
No, it's a check to avoid errors